include/cantera/oneD/StFlow.h

```diff
@@ -292,8 +292,21 @@ class StFlow : public Domain1D
         return m_rho[j];
     }
 
+    //! @deprecated To be removed after %Cantera 3.0. Superseded by isFree()
     virtual bool fixed_mdot();
 
+    //! Retrieve flag indicating whether flow is free
+    //! @since New in %Cantera 3.0
+    bool isFree() const {
+        return m_isFree;
+    }
```

Comment:
I think it would help to elaborate on what it means for the flow to be "free", namely, that the axial velocity at the inlet is unspecified and determined by the solver.

To be honest, the `fixed_mdot` nomenclature is more intuitive to me -- while we call the unstrained flame "freely propagating", we solve it by pinning the flame in place.

Reply:
Regarding nomenclature, `fixed_mdot` is unfortunately ambiguous: it encompasses both strained counter-flow and unstrained burner-stabilized flames. This was the motivation for splitting into `isFree` and `isStrained` (formerly `usesLambda`).